vm_automation/workstationVm.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, configDictionary, logFile = "default.log"):
             self.vmrunExe =     configDictionary['VMRUN_PATH']
             self.vmPath =       configDictionary['VM_PATH']
         except ValueError as e:
-            print "CONFIG FILE DID NOT CONTAIN ALL REQUIRED DATA: " + str(e)
+            print("CONFIG FILE DID NOT CONTAIN ALL REQUIRED DATA: " + str(e))

```

Comment:
`print` should this be logMsg?

Reply:
Ewwwwwww. well, first off, there's no logmsg defined in the workstationVm class, which, well.... is not great and needs to be fixed.  Also, this print happens when the config file is bad, and the config file contains the name of the logfile, so if this fires, we don't have a log file to use.

Reply:
Since this has been extracted as a library module we should not expect print to the terminal.  
Lets remove this for now and throw an error to be caught by the caller.